src/lib/Libwin/util.c

```diff
@@ -975,6 +975,7 @@ revert_impersonated_user()
 			is_user_impersonated = FALSE;
 		else
 			errno = GetLastError();
+			log_err(-1, __func__, "Unable to revert impersonation");
```

Comment:
Why for other functions in this file we didn't modify error logging?

Reply:
Actually this file was there in @raviv1995's commit and there only one file was done.
util.c is under progress. Do you want me to remove this file from current commit ?

Reply:
> Why for other functions in this file we didn't modify error logging?

Actually this file was there in @raviv1995's commit and there only one file was done.
util.c is under progress. Do you want me to remove this file from current commit ?